source/Mlos.Notebooks/SmartCacheOptimization.ipynb

```diff
@@ -641,4 +641,4 @@
  },
  "nbformat": 4,
  "nbformat_minor": 4
-}
```

Comment:
Can you please `git restore` from `main` this file (it's an unnecessary/superficial change).